website/content/en/status/report-2023-04-2023-06/kboot.adoc

```diff
@@ -0,0 +1,55 @@
+=== LinuxBoot FreeBSD on aarch64
+
+Contact: Warner Losh <imp@bsdimp.com>
+
+Links: +
+link:https://www.linuxboot.org/[LinuxBoot Project] URL: link:https://www.linuxboot.org/[]+
+link:https://docs.google.com/presentation/d/1N5Jp6XzYWv9Z9RhhETC-e6tFkqRHvp-ldRDW_9h2JCw/edit?usp=sharing[BSDCan 2023 kboot talk slides] URL: link:https://docs.google.com/presentation/d/1N5Jp6XzYWv9Z9RhhETC-e6tFkqRHvp-ldRDW_9h2JCw/edit?usp=sharing[]
+
+LinuxBoot is an effort to create a clean, robust, auditable and repeatable boot firmware.
+What originally started as a specific project at Google has grown to encompass any boot environment that uses Linux to launch the final operating system.
+Many platfors now support this environment, in some cases it is the only available boot environment.
+In addition, some embedded boxes have a LinuxBoot environment hard-coded that is quite hard to change, and being able to reboot into FreeBSD is desirable.
+
+The old Sony PlayStation 3 port used a boot loader called 'kboot' to boot the FreeBSD port from its Linux kernel (all predating the LinuxBoot project).
+That code has been greatly expanded, made generic with easily replaceable per-architecture plug ins.
+The normal FreeBSD [.filename]#/boot/loader# is built as a Linux binary that reads in the FreeBSD kernel, modules and tunables.
+It places them into memory as if it were running in a pre-boot environment, then loads that image into the Linux kernel with man:kexec_load[2] and does a special reboot to that image.
+For UEFI enabled systems, it passes the UEFI memory table, and pointer to UEFI runtime services to the new kernel.
```

Comment:
```suggestion
For UEFI-enabled systems, it passes the UEFI memory table and pointer to UEFI runtime services to the new kernel.
```

Reply:
If we add a second comma, then it probably needs a third. 

Alternatively: 

> With UEFI-enabled systems, the UEFI memory table and a pointer to UEFI runtime services are passed to the new kernel.